libvast/src/system/source_command.cpp

```diff
@@ -151,6 +153,8 @@ caf::message source_command(const command& cmd, caf::actor_system& sys,
           self->send(importer, subscribe_atom::value, flush_atom::value, self);
         else
           stop = true;
+      } else {
+        VAST_DEBUG(&cmd, "received unexpected DOWN from", msg.source);
```

Comment:
This looks more like an assertion. Can we add one?

Reply:
done.